modules/widgets/src/fullscreen-widget.tsx

```diff
@@ -99,13 +97,17 @@ export class FullscreenWidget implements Widget<FullscreenWidgetProps> {
 
       if (!deepEqual(oldProps.style, props.style, 1)) {
         if (oldProps.style) {
-          Object.entries(oldProps.style).map(([key]) => el.style.removeProperty(key));
-        }
-        if (props.style) {
-          Object.entries(props.style).map(([key, value]) =>
-            el.style.setProperty(key, value as string)
-          );
+          Object.entries(oldProps.style).map(([key]) => {
+            if (key.startsWith('--')) {
+              // Assume CSS variable
+              el.style.removeProperty(key);
+            } else {
+              // Assume camelCase
+              el.style[key] = '';
+            }
+          });
```

Comment:
Let's create a removeStyles as well. This seems useful to apply in all widgets when style has changed